complete/src/main/java/com/example/restservice/service/TokenService.java

```diff
@@ -106,9 +108,18 @@ public TokenDetailResponse createToken(CreateTokenRequest createTokenRequest) {
 
   @Transactional
   public MyTokensResponse getMyTokens() {
-    return new MyTokensResponse(
+    Stream<Token> myTokenStream =
         tokenRepository
             .findByOwnerId(loggedInUserInfo.getUserId())
+            .sorted(
+                new Comparator<Token>() {
+                  public int compare(Token a, Token b) {
+                    return a.getTokenCreationTimestamp().compareTo(b.getTokenCreationTimestamp());
+                  }
+                });
+
+    return new MyTokensResponse(
+        myTokenStream
```

Comment:
same as above, let's not use this extra variable

Reply:
changed